cranelift/assembler-x64/meta/src/instructions/lanes.rs

```diff
@@ -60,9 +60,15 @@ pub fn list() -> Vec<Inst> {
         inst("vblendvpd", fmt("RVMR", [w(xmm1), r(xmm2), r(xmm_m128), r(xmm3)]), vex(L128)._66()._0f3a().w0().op(0x4B).r().is4(), _64b | compat | avx),
 
         // Shuffle lanes in various ways.
+        inst("shufpd", fmt("A", [rw(xmm1), r(align(xmm_m128)), r(imm8)]), rex([0x66, 0x0F, 0xC6]).ib(), _64b | compat | sse2).alt(avx, "vshufpd_b"),
```

Comment:
Note I've added support for shufpd and vshufpd but these instructions did not exist before. Assembler testing passes but there is no filetest and I'd guess this is just dead code. Should I just remove these instructions?